README.md

```diff
@@ -144,6 +144,9 @@ This will build the docker image as `prometheuscommunity/postgres_exporter:${bra
 * `[no-]collector.stat_database`
   Enable the `stat_database` collector (default: enabled).
 
+* `[no-]collector.stat_progress_vacuum`
+  Enable the `stat_progress_vacuum` collector (default: disabled).
```

Comment:
What do you think about making this one enabled by default. Are vacuum stats useful enough for most users?

Reply:
I think enabling this by default is acceptable here.  I'll make it so.

Reply:
This is done.